core/ecschema-metadata/src/Deserialization/JsonProps.ts

```diff
@@ -290,11 +290,16 @@ export type SchemaItemFormatProps = SchemaItemProps & FormatProps;
 
 /**
  * This interface defines properties necessary to support persistence of a set of formats.
- * @beta
+ * @public @preview
  */
 export interface FormatSet {
+  /** The unique name identifier for this format set. */
   name: string;
+  /** The display label for this format set. */
   label: string;
+  /** A [UnitSystemKey]($quantity) that determines the unit system for this format set. */
+  unitSystem: UnitSystemKey;
```

Comment:
UnitSystemKey is still in beta.  We should review the systems listed in this type then move it to public preview at the same time we move the FormatSet interface

Reply:
I've changed the FormatSet interface back to beta, so we won't be moving UnitSystemKey to public preview